libs/api/domains/national-registry-x-road/src/lib/nationalRegistryXRoad.service.ts

```diff
@@ -75,6 +75,20 @@ export class NationalRegistryXRoadService {
     )
   }
 
+  // Returns array representation of nationalIds of individuals children.
+  async getChildrenForsja(
+    user: User,
+    parentNationalId: string,
+  ): Promise<string[] | undefined> {
+    const nationalRegistryApi = this.nationalRegistryApiWithAuth(user)
+    const childrenNationalIds = await nationalRegistryApi
+      .einstaklingarGetForsja({
+        id: parentNationalId,
+      })
+      .catch(this.handle404)
```

Comment:
I see we are doing the same on lines 96-101, we should extract that to a function and reuse that instead of having this duplicated. 

Also in that function (on line 103) we return an empty array if there are no `id's` found. I see you are doing that in your controller, but if you want then you can extract that also to the function and then you can remove it from your controller. 😄